themes/src/main/resources-community/theme/base/admin/theme.properties

```
@@ -1 +1 @@
-locales=ar,ca,de,el,en,es,fa,fr,fi,hr,it,ja,lt,lv,nl,no,pl,pt,pt-BR,ru,sl,sv,uk,zh-CN,zh-TW
+locales=ar,ca,de,el,en,es,fa,fr,fi,hr,it,ja,lt,lv,nl,no,pl,pt,pt-BR,ru,ky,sl,sv,uk,zh-CN,zh-TW
```

The language code are sorted alphabetically, update this and the other theme.properties

```suggestion
locales=ar,ca,de,el,en,es,fa,fr,fi,hr,it,ja,ky,lt,lv,nl,no,pl,pt,pt-BR,ru,sl,sv,uk,zh-CN,zh-TW
```

---

the sequence are sorted alphabetically